index.html

```diff
@@ -47,5 +47,189 @@ <h3>LET'S CONNECT</h3>
+             <li class="titles-item">2015</li>
+           </ul>
+           <p class="works-flex-item works-description">
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+           <div class="works-flex-item">
+             <ul class="lang-container">
+               <li class="lang">html</li>
+               <li class="lang">css</li>
+               <li class="lang">javascript</li>
+             </ul>
+           </div>
+           <div class="works-flex-item">
+             <div class="btn-see">
+               <a href="#" class="btn-see-project">See project</a>
+             </div>
+           </div>
+         </div>
+         <div class="works-flex">
+           <div class="works-flex-item work-image">
+             <img src="snapshoot_portfolio(3).png" class="img-style" alt="tonic works" /> 
+           </div>
+           <div class="works-flex-item">
+             <h3 class="intro-works-text">Multi-Post <br/> Stories</h3>
+           </div>
+           <ul class="works-flex-item titles-container">
+             <li class="list-main">CANOPY</li>
+             <li class="titles-item">Back End Dev</li>
+             <li class="titles-item">2015</li>
+           </ul>
+           <p class="works-flex-item works-description">
+             A daily selection of privately personalized reads; no accounts or sign-ups required.
+           </p>
+           <div class="works-flex-item">
+             <ul class="lang-container">
+               <li class="lang">html</li>
+               <li class="lang">css</li>
+               <li class="lang">javascript</li>
+             </ul>
+           </div>
+           <div class="works-flex-item">
+             <div class="btn-see">
+               <a href="#" class="btn-see-project">See project</a>
+             </div>
+           </div>
+         </div>
+       </section>
+       <section class="aboutme">
+         <div class="aboutme-wrapper">
+           <div class="headline-container">
+             <h2 class="intro-text-about">About<br/> Myself</h2>
+             <p class="intro-about-message">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             <div class="end-message">
+               <h3>LET'S CONNECT</h3>
+             </div>
+           </div>
+
+           <ul class="social-icon">
+             <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+             <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+             <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+             <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+             <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+            </ul>
```

Comment:
- [ ] Great job making the icons as a list, with clickable elements. In order to match the original design better, you should consider displaying them horizontally, not vertically.

Figma design: 
![image](https://user-images.githubusercontent.com/66967274/143580238-cd273bff-a541-448a-b67c-e147f5953fa6.png)
 
Your design:
![image](https://user-images.githubusercontent.com/66967274/143580270-02296193-ff5a-4ddb-847d-4475e9c787eb.png)